repositories/migrations/20240708141735_create_webhook_table.sql

```diff
@@ -2,7 +2,7 @@
 -- +goose StatementBegin
 
 CREATE TABLE IF NOT EXISTS
-      webhooks (
+      webhook_events (
```

Comment:
remember to make sure to run it locally so you don't deviate

Reply:
Everytime I modify this file I run locally a `goose down` + `goose up` to check everything is clean